### PR TITLE
Add 1stKissManga.me to page list

### PR DESCRIPTION
**Pull request by Hanawa02**

### CODE REVIEW

File: src/pages/list.json
Comment (henrik9999) [inline src/pages/list.json:1]: This file is auto generated please don't commit

File: src/pages/FirstKiss/main.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+const URL_MANGA_ID_INDEX = 4;
+
+function isChapterPage(url: string): boolean {
+  return url.split('/')[5].startsWith('chapter');
+}
+
+export const FirstKiss: pageInterface = {
+  name: '1stKissManga',
+  domain: 'https://1stkissmanga.me',
+  languages: ['en-US'],
```
Comment (henrik9999) [inline src/pages/FirstKiss/main.ts:15]: We are using the name so in this case it would be just "English"

**Review by henrik9999 — CHANGES_REQUESTED** [review]

File: src/pages/list.json
Comment (Hanawa02) [reply src/pages/list.json:1]: 💡 Thanks for pointing it out. I removed the changes

File: src/pages/FirstKiss/main.ts (hunk 1 above)
Comment (Hanawa02) [reply src/pages/FirstKiss/main.ts]: Updated

**Pull request merged**